exoctk/contam_visibility/resolve.py

```diff
@@ -1,8 +1,56 @@
+import numpy as np
+
 from exoctk.utils import get_target_data
 
+def deg2HMS(ra='', dec='', round=True):
+  RA, DEC, rs, ds = '', '', '', ''
+  if dec:
+    if str(dec)[0] == '-':
+      ds, dec = '-', abs(dec)
+    deg = int(dec)
+    decM = abs(int((dec-deg)*60))
+    if round:
+      decS = np.round(((abs((dec-deg)*60)-decM)*60), 2)
+    else:
+      decS = (abs((dec-deg)*60)-decM)*60
+    DEC = '{0}{1}:{2}:{3}'.format(ds, deg, decM, decS)
+
+  if ra:
+    if str(ra)[0] == '-':
+      rs, ra = '-', abs(ra)
+    raH = int(ra/15)
+    raM = int(((ra/15)-raH)*60)
+    if round:
+      raS = np.round(((((ra/15)-raH)*60)-raM)*60, 2)
+    else:
+      raS = ((((ra/15)-raH)*60)-raM)*60
+    RA = '{0}{1}:{2}:{3}'.format(rs, raH, raM, raS)
+
+  if ra and dec:
+    return (RA, DEC)
+  else:
+    return RA or DEC

```

Comment:
@jaymedina This is neatly written, but this functionality is already in astropy.coordinates.SkyCoord : https://docs.astropy.org/en/stable/coordinates/.

Reply:
For some reason the code is breaking when I input the RA and DEC coordinates returned using `SkyCoord`. Can we opt to have `SkyCoord` implemented in the next release?